README.md

```diff
@@ -268,16 +178,8 @@ Performance tests are stored in a separate repository which is then run in the p
 You've deleted your unit test db or done something manually, so pytest's cache is confused.
 Run `pytest --cache-clear` to fix your problem.
 
-
-# Extras
-
-This repo comes with a .pre-commit-config.yaml, if you wish to use this do
-the following while in your virtual environment:
-
-    pre-commit install
-
-Once the above is done you will have autoformatting and pep8 compliance built
-into your workflow. You will be notified of any pep8 errors during commits.
+# IDE Setup
+[Python IDE Setup](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-ide-setup.md)
 
 # Database on Paas
 Create db service with:
```

Comment:
We don't use paas any more, and we don't use cf

Reply:
have removed section